README.md

```diff
@@ -149,6 +149,7 @@ Library                               | Since    | Notes
 `github.com/redis/go-redis`           | `v0.9.1` | [Aspect][go-redis]
 `github.com/graph-gophers/graphql-go` | `v0.9.1` | [Aspect][graph-gophers]
 `github.com/graphql-go/graphql`       | `v0.9.1` | [Aspect][graphql]
+`github.com/jackc/pgx`                | `v0.9.4` | [Aspect][pgx]
```

Comment:
You need to add the link to the doc correctly at the end of the file 👍

Reply:
You are right, I left it behind accidentally.

![image](https://github.com/user-attachments/assets/5f94e1c5-41b4-446d-a4bb-be247c5fec61)